SA3/qa_web/views.py

```diff
@@ -51,6 +51,10 @@ def login(request):
         else:
             return render_to_response('qa_web/login.html', context={'form': form})
 
+def logout_view(request):
+    logout(request)
+    return HttpResponseRedirect('../login/')
+    # redirects to log in page once logged out successfully
```

Comment:
I believe the simple `logout` function call you implemented here is enough, as we do not need the session data that goes with the user logged in (it usually gets all flushed away with `logout`). All of the user interaction is handled through actual data being saved on the database. Sick redirect too.